uiowa/uiowa
# Updated Antonio local font for better browser consistency

This fixes the font-weight issue was appearing lighter in Safari without having to switch to Google fonts. 


# How to test

`ddev blt frontend`
Review `uids_base` code changes that switch font paths to use UIDS3 fonts
`ddev blt ds --site=sandbox.uiowa.edu`
Test on chrome and safari and make sure that font weight looks the same.  The current prod site page in Safari should appear as a lighter font weight.
- Test out Antonio usage on https://sandbox.prod.drupal.uiowa.edu/button-page
- Test out Antonio usage on https://sandbox.prod.drupal.uiowa.edu/statistics
- Test out Antonio usage on https://sandbox.prod.drupal.uiowa.edu/banner-backgrounds-w-gradient-dark
